lib/active_fedora/file.rb

```diff
@@ -118,7 +118,7 @@ def original_name
     end
 
     def digest
-      response = metadata.ldp_source.graph.query(:predicate => ::RDF::URI.new("http://fedora.info/definitions/v4/repository#digest"))
+      response = metadata.ldp_source.graph.query(predicate: ActiveFedora::RDF::Fcrepo4.digest)
```

Comment:
Similar to  my comment on the FixityService, how about adding a new method to this class called `digest_url` and have it return `ActiveFedora::RDF::Fcrepo4.digest` so that folks can more easily override this downstream with a URI of their choosing?